moped-database/migrations/1643650785073_alter_table_public_moped_project_add_column_work_assignment_id/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."moped_project" ADD COLUMN "work_assignment_id" text NULL;
```

Comment:
TIL there are no performance benefits of using `varchar(field_length)` in PostgreSQL. Very cool.